unittests/Support/ProgramTest.cpp

```diff
@@ -188,7 +188,12 @@ TEST(ProgramTest, TestExecuteNoWait) {
   ProcessInfo PI1 = ExecuteNoWait(Executable, argv, &envp[0], nullptr, 0,
                                   &Error, &ExecutionFailed);
   ASSERT_FALSE(ExecutionFailed) << Error;
+#ifdef _WIN32
+  // Pid is unsigned on windows.
+  ASSERT_NE(PI1.Pid, 0u) << "Invalid process id";
+#else
   ASSERT_NE(PI1.Pid, 0) << "Invalid process id";
+#endif
```

Comment:
I prefer how upstream solved this: https://github.com/llvm-mirror/llvm/commit/a58e071c716c1f7c639ed63758c83b14a5ee5f61

Reply:
Merged.